src/state/nftMarket/hooks.ts

```diff
@@ -9,3 +13,13 @@ export const useFetchCollections = () => {
     dispatch(fetchNftsFromCollections('0x60935f36e4631f73f0f407e68642144e07ac7f5e'))
   }, [dispatch])
 }
+
+export const useCollectionFromSlug = (slug: string) => {
+  const collections = useSelector((state: State) => state.nftMarket.data.collections)
+  return find(collections, (collection) => slugify(collection.name) === slug)
```

Comment:
Any reason why you are using Lodash's `find`?

Reply:
because it's an object and not an array